css/style.css

```
@@ -16,4 +17,161 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* header section */
+ header{
+     display: flex;
+     justify-content: space-evenly;
+     align-content: center;
+ }
 
+ header img{
+     height: 2.5em;
+     padding: 1rem ;
+   
+}
+
+/* Nav bar */
+
+
+ nav ul{
+     display: flex;
+     list-style: none;
+     justify-content: space-evenly;
+     width: 50rem;
+     padding: 1rem 0rem;
+
+ }
+
+ nav ul a{
+     text-decoration: none;
+     font-size: 1.5rem;
+     color:rgb(197, 189, 189);
+ }
+
+nav ul a:hover{
+    color:rgb(240, 104, 13);
+}
+
+li:first-of-type a{
+    color: rgb(105, 102, 102);
+    font-weight: 500;
+}
+x-special/nautilus-clipboard
+
+/*Main content*/
+.hero{
+  background-position: center;
```

```suggestion
.hero{
  height: 40rem;
  background-image: url("../img/first-background.jpg");
  background-position: center;
```
Instead of putting the background image in HTML, we can put it in here and give the element a specific height